lib/core/versions/latest/BatchWriter.ts

```diff
@@ -49,14 +51,14 @@ export default class BatchWriter implements IBatchWriter {
     const didUniqueSuffixes = batch.map(operation => operation.didUniqueSuffix);
 
     // Construct the 'anchor file'.
-    const anchorFile = {
+    const anchorFileModel: AnchorFileModel = {
       batchFileHash: batchFileHash,
       merkleRoot: encodedMerkleRoot,
       didUniqueSuffixes
     };
 
     // Make the 'anchor file' available in content addressable store.
-    const anchorFileJsonBuffer = Buffer.from(JSON.stringify(anchorFile));
+    const anchorFileJsonBuffer = await AnchorFile.createBufferFromAnchorFileModel(anchorFileModel);
```

Comment:
```suggestion
    const anchorFileCompressedBuffer = await AnchorFile.createBufferFromAnchorFileModel(anchorFileModel);
```

Reply:
Same comment as before, I don't think that the BatchFile necessary knows that the AnchorFile is compressing the buffer.